gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -804,47 +816,43 @@ gst_tensor_filter_transform (GstBaseTransform * trans,
           ml_loge_stacktrace
               ("gst_tensor_meta_info_update_header() has failed to update header for flexible format: invalid metadata or buffer for header is not available. This looks like an internal error of nnstreamer/tensor_filter. Please report to github.com/nnstreamer/nnstreamer/issues. %u'th output buffer has failed to update its header.\n",
               i);
-          goto mem_map_error;
+          retval = GST_FLOW_ERROR;
```

Comment:
Is ```go to done;``` necessary?

Reply:
Before go to `done`, need to unmap `out_info`.

Reply:
Thank you.